clients/client/pom.xml

```diff
@@ -61,6 +40,12 @@
               <goal>shade</goal>
             </goals>
             <configuration>
+              <artifactSet>
+                <includes>
+                  <include>org.tachyonproject:tachyon-common</include>
+                  <include>org.apache.thrift:tachyon.org.apache.thrift</include>
+                </includes>
+              </artifactSet>
               <filters>
```

Comment:
Now that we are actually including specific artifacts (as opposed to everything), you probably can remove the `filters` because neither thrift nor tachyon-common should have signed files.